### PR TITLE
Ajout de Olivier Wulveryck

### PR DESCRIPTION
**Pull request by owulveryck**
Hello,

I have added myself to the baggers.js file.

Thanks

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -17028,6 +17028,44 @@ var data = {
...
+          "abstract": "Le DeepLearning fait naitre des idées folles dans les têtes de beaucoup. Mais les applications vont au delà de l'analyse marketing et de la computer vision. Je vous propose de vous présenter comment j'ai commencé à m'intéresser au machine learning. Des concepts de base à l'implémentation d'un réseau de neurone from scratch en Go.",
+          "tags": [
+            "Machine Learning",
+            "RNN",
+            "Go"
+          ],
+          "lang": [
+            "fr"
+          ]
+        },
```
Comment (dadoonet) [inline baggers.js]: I think this is wrong (comma).

**Review by dadoonet — CHANGES_REQUESTED** [review]
You have a typo I think

**Review by owulveryck — COMMENTED** [review]
I have fixed the typo. Many thanks

**Review by dadoonet — APPROVED** [review]
Bienvenue!

**Pull request merged**